origami.json

```
@@ -1,8 +1,6 @@
 {
-  "description": "Provides a standard way of firing the 'o.DOMContentLoaded' events if/when the equivalent browser-native events fire, to auto initialise other components that are in use.",
-  "origamiType": "module",
-  "origamiCategory": "utilities",
-  "origamiVersion": 1,
+  "origamiType": "component",
```

I think during the spec review sessions we decided to change o-autoinit from a component to a library (I think because it doesn't have css or html), if we want to swap it to a library, we should look at removing the use of obt from the project because obt is for component development/testing
```suggestion
  "origamiType": "library",
```